packages/internal/bridge/sdk/src/lib/validation.test.ts

```diff
@@ -429,4 +431,143 @@ describe('Validation', () => {
       }
     });
   });
+
+  describe('validateGetFee', () => {
+    let bridgeConfig: BridgeConfiguration;
+
+    beforeEach(() => {
+      const voidRootProvider = new ethers.providers.JsonRpcProvider('x');
+      const voidChildProvider = new ethers.providers.JsonRpcProvider('x');
+      bridgeConfig = new BridgeConfiguration({
+        baseConfig: new ImmutableConfiguration({
+          environment: Environment.SANDBOX,
+        }),
+        bridgeInstance: ETH_SEPOLIA_TO_ZKEVM_TESTNET,
+        rootProvider: voidRootProvider,
+        childProvider: voidChildProvider,
+      });
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('does not throw an error when everything setup correctly', async () => {
+      validateGetFee(
+        {
+          action: BridgeFeeActions.DEPOSIT,
+          sourceChainId: ETH_SEPOLIA_TO_ZKEVM_TESTNET.rootChainID,
+          destinationChainId: ETH_SEPOLIA_TO_ZKEVM_TESTNET.childChainID,
+          gasMultiplier: 1.1,
+        },
+        bridgeConfig,
+      );
+    });
+
+    it('throws an error when sourceChainId is not a valid chainId', async () => {
+      expect.assertions(2);
+      try {
+        validateGetFee(
+          {
+            action: BridgeFeeActions.DEPOSIT,
+            sourceChainId: '1293123',
+            destinationChainId: ETH_SEPOLIA_TO_ZKEVM_TESTNET.childChainID,
+            gasMultiplier: 1.1,
+          },
+          bridgeConfig,
+        );
+      } catch (error: any) {
+        expect(error).toBeInstanceOf(BridgeError);
+        expect(error.type).toBe(BridgeErrorType.UNSUPPORTED_ERROR);
```

Comment:
I would have expected this error to be `BridgeErrorType.INVALID_SOURCE_CHAIN_ID`